modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ConfigurationModelValidator.java

```diff
@@ -46,6 +52,30 @@
   public void validate(ExtensionModel model, ProblemsReporter problemsReporter) {
     new ExtensionWalker() {
 
+      @Override
+      protected void onParameter(ParameterizedModel owner, ParameterGroupModel groupModel, ParameterModel model) {
+        model.getType().accept(new MetadataTypeVisitor() {
+
```

Comment:
We should add tests of the validation failing when the methods are not overriden, test that the validation passes when they are overriden, check that we only fail in compile time.